lldb/test/Shell/Recognizer/verbose_trap.test

```diff
@@ -0,0 +1,9 @@
+# RUN: %clang_host -g -O0 %S/Inputs/verbose_trap.cpp -o %t.out
+# RUN: %lldb -b -s %s %t.out | FileCheck %s
+run
+# CHECK: thread #{{.*}}stop reason = __llvm_verbose_trap: Function is not implemented
```

Comment:
We might consider stripping the `__llvm_verbose_trap` from the user-visible output (and replace it with a more high-level text.

Reply:
Done